flux/releaseconfig.json

```
@@ -32,12 +32,12 @@
     "prod_ring2": "1.0.0"
   },
   "linkerd": {
-    "at_ring1": "1.2.0",
+    "at_ring1": "1.1.3",
```

keep at_ring1 at 1.2.0
```suggestion
    "at_ring1": "1.2.0",
```